.travis.yml

```diff
@@ -10,15 +10,177 @@ rust:
 before_script:
   - rustup component add clippy-preview
 
+before_install:
+  - eval "${MATRIX_EVAL}"
+
 script:
   - cargo test -- --test-threads=1
   - cargo clippy
 
+env:
+  global:
+  - RUSTFLAGS="-D warnings"
+
+addons:
+  apt:
+    packages:
+      - make
+      - autoconf
+      - autoconf-archive # for AX_CHECK_COMPILE_FLAG
+      - automake
+      - libtool
```

Comment:
It seems `.travis.yml` can't put these package declarations together, so repeats them multiple times.